app/scripts/constants/sentry-state.ts

```diff
@@ -90,7 +90,6 @@ export const SENTRY_BACKGROUND_STATE = {
     termsOfUseLastAgreed: true,
     timeoutMinutes: true,
     trezorModel: true,
-    usedNetworks: true,
```

Comment:
We will need a migration to remove this as well, otherwise it will remain in state for all users.

Reply:
@Gudahtt Yes I just made a commit for the migration https://github.com/MetaMask/metamask-extension/pull/28813/commits/b36afd2cffcfb9ca3bb2263367059d7036519591 I left a console.log I will remove it in the upcoming commit

Reply:
Great, thanks!

We could even leave the log in. I find logs helpful for migrations. They only happen once, they're not too noisy, and that can help debugging errors on Sentry or reports via CS.